source/installation/install-kafka-cluster.txt

```diff
@@ -0,0 +1,146 @@
+
+   .. step:: Install Relational Migrator Sink Connect Plugin
+
+      1. Download the latest version of the ``kafka-connect.jar`` file.
+
+         To download the latest ``kafka-connect.jar`` file, see the `Migrator Installer 
+         Repository <https://migrator-installer-repository.s3.ap-southeast-2.amazonaws.com/index.html>`__.
+
+      #. Copy the ``.jar`` file to the ``kafka-connect-migrator`` directory.
+
+         The new ``kafka-connect-migrator`` directory is found at Kafka Connect’s `plugin.path 
+         <https://kafka.apache.org/documentation/#connectconfigs_plugin.path>`__. For example: 
+         ``/kafka/connect/kafka-connect-migrator``. 
+
+      #. Restart your Kafka Connect cluster. 
+
+         To know if your plugin is successfully registered, you must see the following 
+         log statement in the ``kafka-connect`` start up logs:
+
+         .. code-block::
+            :copyable: false
+
+            INFO || Added plugin 'com.mongodb.migrator.kafka.connect.MigratorSinkConnector'
+
+   .. step:: Install Relational Migrator
+
+      From the `download center <https://www.mongodb.com/try/download/relational-migrator>`__, 
+      select the appropriate Relational Migrator installer.
+
+   .. step:: Run Relational Migrator in ``kafka-integrated`` mode
+
+      To run in ``kafka-integrated`` mode, update the following configurations in your 
+      ``user.properties`` file.
+
+      Your ``user.properties`` file is located in an OS specific location. For more 
+      information, see :ref:`Relational Migrator File Locations <file-location>`.
+
+      .. list-table:: 
+         :header-rows: 1
+         :width: 35 15 50
+
+         * - Property Name
+           - Default 
+           - Description 
+
+         * - ``spring.profiles.active``
+           - local
+           - Relational Migrator can be run in the following profile: 
+
+             - ``kafka``: Remote server using embedded server deployment.
+             - ``confluent``: Remote server using Confluent Cloud for managed Kafka
+
+             Set the profile value to ``kafka``. 
+
+         * - ``server.port``
+           - 8278
+           - The port at which the Relation Migrator UI is available.
+
+         * - ``migrator.kafka.bootstrap.servers``
+           - No default
+           - This is a list of ``host:port`` pairs of your pre-existing kafka cluster. 
+             
+             For more information, see `Worker Configuration Properties 
+             <https://docs.confluent.io/platform/current/connect/userguide.html#worker-configuration-properties-file>`__. 
+
+         * - ``migrator.kafka.connect.url``
+           - No default 
+           - The Kafka Connect host url using the ``{host}:{port}`` format. 
+
+         * - ``migrator.kafka.connect.metrics.jmx.service.url``
+           - No default
+           - The Kafka Connect ``jmx`` url. 
+
+Learn More
+----------
+
```

Comment:
[For Reviewer]

These are currently causing build errors but will be resolved when https://github.com/mongodb/docs-relational-migrator/pull/96 is merged